changes/2020-06-09_wrapping-key-identifiers/change.md

```diff
@@ -0,0 +1,151 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Clarify Wrapping Key Identifiers
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                                                                             |
+| --------------------------------------------------------------------------------------------------- |
+| [Encrypted Data Key](../../framework/structures.md#encrypted-data-key)                              |
+| [AWS KMS Keyring](../../framework/kms-keyring.md)                                                   |
+| [Raw AES Keyring](../../framework/raw-aes-keyring.md)                                               |
+| [Raw RSA Keyring](../../framework/raw-rsa-keyring.md)                                               |
+| [Keyring Decryption Contract](https://github.com/awslabs/aws-encryption-sdk-specification/pull/131) |
```

Comment:
@acioc had the idea of making these permanent links to the specific commit before the change.

Reply:
I think there's some ongoing discussion of this on https://github.com/awslabs/aws-encryption-sdk-specification/issues/137